File: 1 - Iniciante/C99/BEE1094.c
#include <stdio.h>

int main(void){
	int casos, qtd, total=0;
	int i, c=0, r=0, s=0;
	char tipo;
	
	scanf("%d", &casos);
	
	for (i = 1; i<=casos; i++){
		
		scanf("%d %c", &qtd, &tipo);
		
		total += qtd;
		
		if (tipo == 'C'){
			c += qtd;
		} else if (tipo == 'R'){
			r += qtd;
		} else s += qtd;
	}
	
	printf("Total: %d cobaias\n", total);
	printf("Total de coelhos: %d\n", c);
	printf("Total de ratos: %d\n", r);
	printf("Total de sapos: %d\n", s);
	printf("Percentual de coelhos: %.2f %%\n", (float) (c * 100) / total);
	printf("Percentual de ratos: %.2f %%\n", (float) (r * 100) / total);
	printf("Percentual de sapos: %.2f %%\n", (float) (s * 100) / total);
	
	return 0;
}

File: 1 - Iniciante/Python 3.11/BEE1094.py
cases = int(input())

total = 0
sapos, ratos, coelhos = 0, 0, 0
for i in range(cases):
    qt, tp = input().split(' ')
    qt = int(qt)

    match (tp):
        case 'S':
            sapos += qt
        case 'R':
            ratos += qt
        case 'C':
            coelhos += qt
    
    total += qt

print(f'Total: {total} cobaias')
print(f'Total de coelhos: {coelhos}')
print(f'Total de ratos: {ratos}')
print(f'Total de sapos: {sapos}')
print(f'Percentual de coelhos: {float((coelhos * 100)/total):.2f} %')
print(f'Percentual de ratos: {float((ratos * 100)/total):.2f} %')
print(f'Percentual de sapos: {float((sapos * 100)/total):.2f} %')